src/MultiplayerMod/Game/Chores/ChoreList.cs

```diff
@@ -16,58 +17,7 @@ public static class ChoreList {
 
     public static readonly Dictionary<Type, ChoreSyncConfig> Config =
```

Comment:
Btw, could we keep the list if we don't remove the class? It will help me to transfer configs to new approach.